server/chat-plugins/chatlog.ts

```diff
@@ -63,6 +63,10 @@ export class LogReaderRoom {
 	}
 
 	async listMonths() {
+		if (roomlogTable) {
+			const dates = await roomlogTable.query()`SELECT DISTINCT month FROM roomlog_dates WHERE roomid = ${this.roomid}`;
+			return dates.map(x => (x as any).month);
+		}
```

Comment:
```suggestion
		if (roomlogTable) {
			const dates = await roomlogTable.query<any>()`SELECT DISTINCT month FROM roomlog_dates WHERE roomid = ${this.roomid}`;
			return dates.map(row => row.month);
		}
```

Honestly you should probably just make a `roomlogDatesTable` but if you're not gonna do it the TypeScript way, you should use the type parameter rather than casting.

(Same change applies to `listDays`)

Reply:
Right, forgot we had that. Done!